bundles/io/org.eclipse.smarthome.io.net.security/README.md

```diff
@@ -0,0 +1,27 @@
+# NetworkSecurityTls
+
+This service is responsible for providing SSLContexts that can be used all over ESH
+for server applications like webservlets, Mqtt servers or any extension that need to
+provide for instance a REST interface.
+
+The SSLContext is configured to use either the user provided java keystore file
+(`../etc/default.keystore`) or if none is provided to generate a self-signed certificate
```

Comment:
```
String filenameToJavaKeyStore = ConfigConstants.getUserDataFolder() + File.separator
                + contextConfig.keystoreFilename;
File userDir = new File(ConfigConstants.getUserDataFolder());
if (!userDir.exists()) {
      userDir.mkdirs();
}
```
This is how it is used. The JavaDoc might be confusing, it should read: ESH_HOME_DIR/etc/default.keystore I guess.